video/out/opengl/egl_helpers.c

```diff
@@ -100,8 +101,32 @@ static void *mpegl_get_proc_address(void *ctx, const char *name)
     return p;
 }
 
+struct egl_opts {
+    int config_id;
+    int output_format;
+};
+
+#define MAKE_FORMAT(r, g, b, a) (r) << 24 | (g) << 16 | (b) << 8 | (a)
+
+#define OPT_BASE_STRUCT struct egl_opts
+const struct m_sub_options egl_conf = {
+    .opts = (const struct m_option[]) {
+        {"egl-config-id", OPT_INT(config_id)},
+        {"egl-output-format", OPT_CHOICE(output_format,
+            {"auto", 0},
+            {"rgb8", MAKE_FORMAT(8, 8, 8, 0)},
+            {"rgb10", MAKE_FORMAT(10, 10, 10, 0)},
+            {"rgba8", MAKE_FORMAT(8, 8, 8, 8)},
+            {"rgb10_a2", MAKE_FORMAT(10, 10, 10, 2)},
+            {"rgb16f", MAKE_FORMAT(16, 16, 16, 0)},
+            {"rgba16f", MAKE_FORMAT(16, 16, 16, 16)})},
```

Comment:
the `f` here is misleading since EGL does not specifically deal in integers or floats

Reply:
fixed in 33ccc693efc46fbe8c60f818a19a3eb4992a5c1c